### PR TITLE
Add arsenal zeus modules

### PR DESCRIPTION
**Pull request by alganthe**
**When merged this pull request will:**
- Add an add full arsenal curator module
- Add a remove arsenal curator module

### CODE REVIEW

File: addons/zeus/stringtable.xml [hunk 1]
```diff
@@ -664,5 +664,13 @@
             <Japanese>無効な半径が入力されました</Japanese>
             <Polish>Wpisano nieprawidłowy promień</Polish>
         </Key>
+        <Key ID="STR_ACE_Zeus_moduleAddFullArsenal_displayName">
+            <English>Add full arsenal to object</English>
+            <French>Ajouter arsenal complet à l'objet</French>
+        </Key>
+        <Key ID="STR_ACE_Zeus_moduleRemoveArsenal_displayName">
+            <English>Remove arsenal from object</English>
+            <French>Retirer arsenal de l'object</French>
```
Comment (bux578) [inline addons/zeus/stringtable.xml]: *cough, cough*

Comment (alganthe) [inline addons/zeus/stringtable.xml]: woops, I didn't save in tabler after I fixed that 👍

File: addons/zeus/stringtable.xml [hunk 2]
```diff
@@ -664,5 +664,13 @@
...
+            <English>Add full arsenal to object</English>
+            <French>Ajouter arsenal complet à l'objet</French>
+        </Key>
+        <Key ID="STR_ACE_Zeus_moduleRemoveArsenal_displayName">
+            <English>Remove arsenal from object</English>
+            <French>Retirer arsenal de l'objet</French>
+        </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/zeus/stringtable.xml]: newline at EOF please.

File: addons/zeus/functions/fnc_moduleAddArsenal.sqf [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+if !(isNull _object) then {
+    switch (false) do {
+        case !(isPlayer _object): {
+            [LSTRING(OnlyNonPlayer)] call _fnc_errorAndClose;
+        };
+        case (alive _object): {
+            [LSTRING(OnlyAlive)] call _fnc_errorAndClose;
+        };
+    };
```
Comment (commy2) [inline addons/zeus/functions/fnc_moduleAddArsenal.sqf]: could this be changed to `switch (true)`? It already inverts the `isPlayer`.
Same below

**Comment by commy2** [comment]
lgtm aside from my two comments. haven't tried ingame though.

**Comment by madpat3** [comment]
question:
how does it work? 
as zeus, i placed a box. then i placed the zeus module (add full ace arsenal) right over it. nothing happens.

**Comment by bux** [comment]
@madpat3 please don't revive old tickets.
questions here please: https://slackin.ace3mod.com

**Pull request merged**